### DISCUSSION TITLE
Speech impediment recognition

### DISCUSSION THREAD

**Discussion by ytain** [coqui-ai/STT#2292] [General]
Is it possible to use this project to make an android app kinda similar to Project Relate, purely offline? 
Of course the model training would be done on a computer with a preset of phrases and scoring for it.

**Comment by wasertech** [comment]
There is [this example project to stream the microphone of an android device](https://github.com/coqui-ai/STT-examples/tree/r1.0/android_mic_streaming). It should work with [any model available in the zoo](https://coqui.ai/models).
